src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -132,7 +132,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $server = new WebServer();
             $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
 
-            $io->success(sprintf('Server listening on http://%s', $config->getAddress()));
+            $message = sprintf('Server listening on http://%s', $config->getAddress());
+            if ('' !== $extra = $config->getExtraAddress()) {
+                $message = sprintf('%s (http://%s)', $message, $extra);
```

Comment:
Not sure if we can hardcode `http://` here. What if we are using `https://` in the local environment? Maybe it's better to detect that and add it in the `getExtraAddress()` method.